srcpkgs/libhandy1/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'libhandy1'
 pkgname=libhandy1
-version=1.5.0
+version=1.6.1
 revision=1
 wrksrc="libhandy-${version}"
 build_style=meson
 build_helper="gir"
 configure_args="-Dexamples=true -Dtests=true $(vopt_bool gtk_doc gtk_doc)
  $(vopt_bool gir vapi) -Dglade_catalog=$(vopt_if glade enabled disabled)
  -Dintrospection=$(vopt_if gir enabled disabled)"
-hostmakedepends="glib-devel pkg-config $(vopt_if gtk_doc gtk-doc) $(vopt_if gir vala-devel)"
+hostmakedepends="glib-devel pkg-config $(vopt_if gtk_doc 'gtk-doc gi-docgen') $(vopt_if gir vala-devel)"
```

Comment:
```suggestion
hostmakedepends="glib-devel pkg-config $(vopt_if gtk_doc gi-docgen) $(vopt_if gir vala-devel)"
```
or is there a reason why you also added gtk-doc?

Reply:
Thanks, tbh I can't remember why I thought it was necessary to leave it in.